play.py

```diff
@@ -1,24 +1,21 @@
-#!/usr/bin/env python
+# let the model play mariokart while being watched
 
 from utils import resize_image, XboxController
-import tensorflow as tf
-import model
 from termcolor import cprint
 import gym
 import gym_mupen64plus
-
+from keras.models import load_model  # if wanted
+from train import creation_model
+import numpy as np
 
 # Play
-class Actor(object):
+class Actor:
```

Comment:
can you add the `object` back please? I don't fully understand the benefits but some bot told us this way is better and we're listening :)